src/components/shop/scrollable-list/scrollable-list-buttons.js

```diff
@@ -71,11 +74,13 @@ const getButtonState = (scene, item, title) => {
     states.push(isButtonCta ? "actioned" : "cta");
     states.push(isItemEquippable(item) ? "equippable" : "consumable");
     states.push(isItemInStock(item) ? "available" : "unavailable");
+    states.push(isItemLocked(item) ? "locked" : "unlocked");
     return states;
 };
 
-const isItemEquippable = item => item.slot;
-const isItemInStock = item => item?.qty > 0;
+const isItemEquippable = item => Boolean(item.slot);
+const isItemInStock = item => Boolean(item?.qty > 0);
+const isItemLocked = item => Boolean(item.state === "locked");
```

Comment:
don't lock this to a name of a state, instead check whether the button should be enabled/disabled in that state

eg. in select screen we do something like this, where enabled: true/false determines whether the button should be active or not. this allows the state to have any name and allows you to define multiple locked states.

I guess then your activeState would be - enabled/disabled

```
    states: {
        locked: {
            enabled: false,
        }
}
```

Reply:
1e1df88 re getting rid of the hardcoded 'locked'. Re overlap between states / activeStates, states is a list of state labels on the scrollable list button, whereas activeStates is a list of state labels to use as a filter ('if all these state labels are on the button, add this overlay.') Probably needs a rename, I'll have a think.

Reply:
Ok- in 3a799be, activeStates is now activeInStates, which is hopefully clearer.